src/components/landing/EventCard/IconsEventCard/IconsEventCard.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./IconsEventCard.scss";
+
+const IconsEventCard = ({ styleForIcon }) => {
+  const MAPPER = [
+    {
+      id: 1,
+      nameControl: "favorite",
+      styles: `heart__icon add-favorite-${styleForIcon}`,
+      todo: "TODO: CREATE FUNCTIONAL FOR ADD FAVORITE EVENT",
+    },
+    {
+      id: 2,
+      nameControl: "joinToEvent",
+      styles: `join-to-event__icon join-to-event-${styleForIcon}`,
+      todo: "TODO: CREATE FUNCTIONAL FOR JOIN TO EVENT",
+    },
+  ];
+
+  return MAPPER.map(({ id, nameControl, styles, todo }) => (
+    <label key={id} htmlFor={nameControl} className={styles}>
+      <button
+        type="button"
+        name={nameControl}
+        id={nameControl}
+        onClick={() => {
+          console.log(todo);
+        }}
+      >
+        {" "}
```

Comment:
What does this button actually doing? It prints no text and call console.log on click

Reply:
This button has a background icon and in the future this button should save a favorite event. This button doesn't print text because {""} fixes eslint error _A form label must be associated with a control_